test/functional/build_helpers/cmake_ios_cross_build_test.py

```diff
@@ -1,40 +0,0 @@
-import platform
-import textwrap
-
-import pytest
-
-from conan.test.utils.tools import TestClient
-
-
-@pytest.mark.xfail(reason="This was using legacy cmake_find_package generator, need to be tested "
-                   "with modern, probably move it to elsewhere")
-@pytest.mark.skipif(platform.system() != "Darwin", reason="Only for Apple")
-@pytest.mark.tool("cmake")
-def test_cross_build_test_package():
```

Comment:
this test is already covered by other test ``test_iphoneos_crossbuild`` in https://github.com/conan-io/conan/blob/c55ab23d01f1336c850816e07c68ae19704201d6/test/functional/toolchains/cmake/cmakedeps/test_apple_frameworks.py#L511